### PR TITLE
Add vibe buttons

### PR DESCRIPTION
**Pull request by GabeBenjamin**
Add vibe functionality

Make it so when you click on a vibe it throws the relevant gif up in the air.

### CODE REVIEW

File: assets/js/vibe.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  timeline.call(() => vibeEl.remove());
+}
+
+/**
+ * Create the associate vibe element and add it to the DOM
+ *
+ * @param {Element} target
+ */
+function vibeClicked(target) {
+  const vibeSrc = vibeMap[target.id];
```
Comment (rglassett) [inline assets/js/vibe.js]: I'd probably have gone with a `data-vibe-src` attribute, using ids in this way feels awkward and fragile.

File: assets/js/vibe.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+const images = new Array();
+function preloadVibeImages() {
+  let i = 0;
+  Object.values(vibeMap).forEach((vibeSrc) => {
+    const tempImage = new Image();
+    tempImage.src = vibeSrc;
+    images.push(tempImage);
+  });
+}
+preloadVibeImages();
```
Comment (rglassett) [inline assets/js/vibe.js]: I'd say this function is fine to export, but I'd probably move the invocation to `app.js` or a script tag in `index.html`--doesn't seem right for a library to make network calls while it loads.

**Review by rglassett — APPROVED** [review]
Your GIF game is impeccable, as always!

File: assets/js/vibe.js (hunk 1 above)
Comment (GabeBenjamin) [reply assets/js/vibe.js]: I like that suggestion! Done!

File: assets/js/vibe.js (hunk 2 above)
Comment (GabeBenjamin) [reply assets/js/vibe.js]: Moved to app.js

**Pull request merged**